persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -58,6 +58,7 @@ private PersistToolsConstants() {}
     public static final String PLATFORM = "java11";
     public static final String PROPERTY_KEY_PATH = "path";
     public static final String MYSQL_CONNECTOR_NAME_PREFIX = "mysql-connector";
+    public static final String SCHEMA_FILE_NAME = "schema";
```

Comment:
Shall we rename the default file name to `model.bal` instead of `schema.bal`

Reply:
fixed in [455671d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/455671d8e9c17171f07009fb84c92ea9438c3abf)